Dockerfile

```diff
@@ -1,12 +1,18 @@
-FROM golang:1.5
+FROM golang:1.8
 RUN apt-get update && apt-get install -yqq aspell aspell-en libaspell-dev tesseract-ocr tesseract-ocr-eng imagemagick optipng exiftool libjpeg-progs webp
 ADD docker/meme.traineddata /usr/share/tesseract-ocr/tessdata/meme.traineddata
 ADD docker/imagemagick_policy.xml /etc/ImageMagick-6/policy.xml
 RUN mkdir -p /etc/mandible /tmp/imagestore
 ENV MANDIBLE_CONF /etc/mandible/conf.json
-ENV GO15VENDOREXPERIMENT 1
 ADD . /go/src/github.com/Imgur/mandible
 WORKDIR /go/src/github.com/Imgur/mandible
+
+RUN wget "https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-1.0.0.tar.gz"
```

Comment:
Have you tried the GPU version? We've had pretty good luck using [open_nsfw](https://github.com/yahoo/open_nsfw)'s script with CUDA drivers, and Amazon recently released an AMI which included a bunch of ML frameworks pre-installed, making setup a bit easier... https://aws.amazon.com/marketplace/pp/B01M0AXXQB

Would be pretty sweet to get this running on all uploads.